lib/steps/setupRedirects.js

```diff
@@ -50,6 +56,12 @@ const setupRedirects = () => {
     });
   });
 
+  // This redirect takes care of this issue: https://github.com/netlify/next-on-netlify/issues/43
+  // where the page chunk for a root level catch-all is served incorrectly to the client.
+  // Add last because it's least specific.
+  if (nextRedirects.length >= 1 && hasCatchAllRoute)
+    redirects.push("/_next/*  /_next/static/*  200");
```

Comment:
This won't work, right?

If I have a redirect with `/*  /:splat  200`, any redirects after that will be ignored. So adding this redirect at the very end won't work.